nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkService.java

```diff
@@ -18,16 +18,23 @@
  */
 package org.nuxeo.ecm.core.bulk;
 
-import java.util.UUID;
-
 /**
- * API to manage Bulk Operation Framework.
+ * API to manage Bulk Computation.
  *
  * @since 10.2
  */
 public interface BulkService {
 
-    BulkStatus runOperation(BulkCommand command);
+    /**
+     * Submits the {@link BulkCommand} for computation and get the initial status containing the bulk id.
+     *
+     * @param command the command to submit
+     * @return the initial bulk status
+     */
```

Comment:
Submits a {@link BulkCommand} that will be processed asynchronously.
@return a unique   bulk command identifier